content/docs/reference/spec/migration/buildpack-api-0.4-0.5.md

```diff
@@ -0,0 +1,39 @@
++++
+title="Buildpack API 0.4 -> 0.5"
++++
+
+<!--more-->
+
+This guide is most relevant to stack and buildpack authors. 
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.5) for buildpack API 0.5 for the full list of changes and further details.
+
+## Stack author
+
+### Stack ID
+Stack ID requirements have been removed from the buildpack specification.
+Each stack must indicate either a compatible stack and provide a valid stack ID, or compatibility with any stack by specifying the special value `*`.
+ 
+## Buildpack author
+
+### Character restrictions for process types
+`type` has been restricted to only contain numbers, letters, and the characters ., _, and -, and now symlinks on both linux and windows filesystems can be created.
+
+### Cached layers
+Cached layers is a new layer type. A buildpack may specify that a `<layers>/<layer>/` directory is a cached layer by placing `cache = true` in `<layers>/<layer>.toml`.
+If a cache is provided, the lifecycle will store all cached layers after a successful build.
```

Comment:
We had cached layers before, I'm not sure I understand what is changing.

Reply:
I'm sorry, it's a bad phrasing. The [spec](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.5) indicates:
```
Adds "Cached Layers" to "Layer Types" to improve clarity
```
I'll reword this section.

Reply:
This is a non-functional change, just making the spec clearer. So I think we could take it out of the guide.

Reply:
@natalieparellano, I agree with you so I've just pushed a commit that deleted it.
@jromero, @ekcasey, please let me know if you think I should add it back.
In the meantime, I'm going to mark this conversation as resolved.